.github/workflows/ci.yml

```diff
@@ -1,19 +1,33 @@
-name: build
+name: Build Status
 on:
+  workflow_dispatch:
   push:
+    branches: [master]
   pull_request:
-    types: [opened, reopened]
+    branches: [master]
 
 jobs:
-  test:
+  build:
     runs-on: ubuntu-latest
+    env:
+      CI: true
+
     strategy:
       matrix:
-        version: [8, lts/*, latest]
+        node-version: [8.x, 10.x, 12.x, 14.x, 16.x, 18.x]
```

Comment:
- for conciseness, it should suffice to specify just `n` instead of `n.x`
- I would like to retain the `lts/*` and `latest` aliases, so that in the event that no one is available to update the list of versions, we have some assurance that we are testing against actively maintained versions of node

Reply:
For reference, I think that this might be good to keep in mind: https://github.com/actions/setup-node/issues/26#issuecomment-1148828915